docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -386,3 +386,34 @@ You can control various behaviors of restore with lock file as described below:
 | `-LockedMode` | `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
 | `-ForceEvaluate` | `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
 | `-LockFilePath` | `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
+
+### AssetTargetFallback
+
+The `AssetTargetFallback` property lets you specify additional compatible framework versions for projects that your project references and NuGet packages that your project consumes.
+
+If you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's target framework, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each target framework that's specified in `AssetTargetFallback`.
+When a `project` or a `package` is referenced through `AssetTargetFallback`, the [NU1701](../reference/errors-and-warnings/NU1701.md) warning will be raised.
+
+Refer to the below table for examples of how `AssetTargetFallback` affects compatibility.
+
+| Project framework | AssetTargetFallback | Package frameworks | Result |
+|-------------------|---------------------|--------------------|--------|
+| .NET Framework 4.7.2 | | .NET Standard 2.0, .NET Standard 1.6 | .NET Standard 2.0 |
+| .NET Core App 3.1 | | .NET Standard 2.0, .NET Framework 4.7.2 | .NET Standard 2.0 |
+| .NET Core App 3.1 | | .NET Framework 4.7.2, .NET Framework 4.7.1 | Incompatible, fail with [`NU1202`](../reference/errors-and-warnings/NU1202.md) |
+| .NET Core App 3.1 | net472;net471;net462;net461 | .NET Framework 4.7.2, .NET Framework 4.7.1 | .NET Framework 4.7.2 with [`NU1701`](../reference/errors-and-warnings/NU1701.md) |
```

Comment:
I think having more than 1 package framework in these examples makes adds unnecessary complexity. In all 4 examples, the 2nd framework in the "package frameworks" list does not come into play. Similarly the long list of frameworks in the last example's AssetTargetFallback value doesn't add clarification. If you want to show that it can be a list, then 2 frameworks is sufficient, in my opinion.

Reply:
I'll do another pass and clean it up.

Reply:
I'll keep the 2 frameworks in the 2nd example, since it points out the precedence. 

Cleaned up the rest.